lib/test_suite.py

```diff
@@ -269,17 +269,18 @@ def run_test(self, test, server, inspector):
         color_stdout(just_and_trim(conf, 15) + ' ', schema='test_var')
 
         if self.is_test_enabled(test, conf, server):
-            short_status, result_checksum = test.run(server)
+            short_status, result_checksum, duration = test.run(server)
```

Comment:
I'll cite my past question:

> How it works with reruns? Please, clarify.

Reply:
Running pre-broken box/cfg.test.lua test showed the following result statistics:
```
Durations of the failed tasks:
*   0.50 engine/misc.test.lua:memtx                                   
```
After added checksum to box/suite.ini to enable rerun for 10 times by default, got:
Durations of the failed tasks:
*   0.50 engine/misc.test.lua:memtx                                   
```
This result showed the latest run result from reruns.